test/unit/clusterstatusmgtd/test_clusterstatusmgtd.py

```diff
@@ -68,3 +80,393 @@ def test_sleep_remaining_loop_time(mocker, loop_start_time, loop_end_time, loop_
     elif expected_sleep_time == 0:
         sleep_mock.assert_not_called()
     datetime_now_mock.now.assert_called_with(tz=timezone.utc)
+
+
+def test_run_command():
+    pass
+
+
+def test_write_json_to_file():
+    # tested in test_call_update_event
+    pass
+
+
+class TestComputeFleetStatus:
+    """Class to test ComputeFleetStatus."""
+
+    def test_transform_compute_fleet_data(self):
+        """Tested in test_call_update_event."""
+        pass
+
+    def test_is_start_in_progress(self):
+        """Tested in test_manage_cluster_status."""
+        pass
+
+    def test_is_stop_in_progress(self):
+        """Tested in test_manage_cluster_status."""
+        pass
+
+    def test_is_protected_status(self):
+        """Not tested, not used."""
+        pass
```

Comment:
do we want to keep these empty tests?

Reply:
just a placeholder to remember those are tested in other methods